e2e/registration.spec.ts

```diff
@@ -1,124 +1,139 @@
 import { expect } from '@playwright/test';
 import { faker } from '@faker-js/faker';
 import { test } from './fixtures/test-options.ts';
-import { getProjects } from './page-objects/helperBase.ts';
+import { getProjects, getTestData } from './page-objects/helperBase.ts';
 import type { Project } from './page-objects/helperBase.ts';
 import type { APIRequestContext } from '@playwright/test';
 
+const countriesToTest = ['us', 'ca', 'pl'];
+
 const projects = getProjects().filter((project: Project) => project.env !== 'prod');
+const phoneByCountry: [string, string][] = Object.entries(
+  getTestData('phone-by-country'),
+);
+
+const testedPhoneByCountry = phoneByCountry.filter((countryPhone) =>
+  countriesToTest.includes(countryPhone[0]),
+);
```

Comment:
You can just use chaining

```
const testedPhoneByCountry = Object.entries(getTestData('phone-by-country'))
  .filter((countryPhone) => countriesToTest.includes(countryPhone[0]));
```

Reply:
firstly i had just phoneByCountry
so it's just about fast coding 

chaining could optimize HEAP storage for objects being stored. huge phoneByCountry will not be stored then

ok

Reply:
fixed